platforms/android/library/src/main/java/io/element/android/wysiwyg/utils/ViewLazyViewModelExtension.kt

```diff
@@ -22,6 +29,18 @@ inline fun <reified VM : ViewModel> View.viewModel(
                         ?: modelClass.newInstance()
                 }
             }
-        }
+        },
     )
 }
+
+fun Context.getViewModelStoreOwner(): ViewModelStoreOwner {
```

Comment:
(sorry for the late remark). Could be private (or internal), but may be useful for library users, hence this is not the aim of the library to expose this.

Reply:
Fixed.

Reply:
Actually, I just remembered why it wasn't private/internal. It seems like it can't be when used from an inline function. I'll revert the commit.

Reply:
Strange, but OK